### ISSUE TITLE
Duplicate column doesn't work for inner section widget.

### ISSUE THREAD

**Issue by WebFerrante** [elementor/elementor#9824]
Duplicate column doesn't work for inner section widget.

## Prerequisites

- [ x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.

## Description
When I trying to duplicate a column of inner section, it doesn't work.
Just with Elementor 2.8.0

## Steps to reproduce

http://somup.com/cql6DFfykG

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
`
### wp-core ###

version: 5.3
site_language: it_IT
user_language: it_IT
timezone: +00:00
permalink: /%year%/%monthnum%/%day%/%postname%/
https_status: true
user_registration: 0
default_comment_status: open
multisite: false
user_count: 2
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /home/rw3lnlsn/public_html
wordpress_size: loading...
uploads_path: /home/rw3lnlsn/public_html/wp-content/uploads
uploads_size: loading...
themes_path: /home/rw3lnlsn/public_html/wp-content/themes
themes_size: loading...
plugins_path: /home/rw3lnlsn/public_html/wp-content/plugins
plugins_size: loading...
database_size: loading...
total_size: loading...

### wp-active-theme ###

name: GestCAV (onzo-progression-child)
version: undefined
author: ProgressionStudios
author_website: (undefined)
parent_theme: Onzo Progression (onzo-progression)
theme_features: wc-product-gallery-slider, post-thumbnails, automatic-feed-links, post-formats, title-tag, menus, woocommerce, widgets
theme_path: /home/rw3lnlsn/public_html/wp-content/themes/onzo-progression-child

### wp-parent-theme ###

name: Onzo Progression (onzo-progression)
version: 1.7
author: Progression Studios
author_website: http://progressionstudios.com/
theme_path: /home/rw3lnlsn/public_html/wp-content/themes/onzo-progression

### wp-themes-inactive (3) ###

GestCAV: version: 1.0.0, author: Jivan Ferrante
Hello Elementor: version: 2.2.1, author: Elementor Team
Twenty Twenty: version: 1.0, author: Team di WordPress

### wp-plugins-active (12) ###

BackupBuddy: version: 8.4.19.0, author: iThemes
Boosted Elements: version: 2.7, author: Progression Studios
CMB2: version: 2.6.0, author: CMB2 team
Contact Form 7: version: 5.1.6, author: Takayuki Miyoshi
Duplicate Page: version: 4.0, author: mndpsingh287
Easy Google Fonts: version: 1.4.4, author: Titanium Themes
Elementor: version: 2.8.0, author: Elementor.com
Elementor Pro: version: 2.7.3, author: Elementor.com
Nested Pages: version: 3.1.7, author: Kyle Phillips
One Click Demo Import: version: 2.5.2, author: ProteusThemes
Progression Theme Elements - Onzo: version: 1.1, author: Progression Studios
Wordfence Security: version: 7.4.2, author: Wordfence

### wp-plugins-inactive (2) ###

Swift Performance Lite: version: 2.1.2, author: SWTE
WP Instagram Widget: version: 2.0.3, author: Scott Evans

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Non disponibile
imagemagick_version: Non disponibile
gd_version: bundled (2.1.0 compatible)
ghostscript_version: 8.70

### wp-server ###

server_architecture: Linux 2.6.32-954.3.5.lve1.4.59.el6.x86_64 x86_64
httpd_software: Apache
php_version: 7.3.5 64bit
php_sapi: cgi-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 256M
max_input_time: -1
upload_max_size: 2M
php_post_max_size: 8M
curl_version: 7.62.0 OpenSSL/1.0.2k
suhosin: false
imagick_availability: false
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 5.6.43-cll-lve
client_version: 5.6.44

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/rw3lnlsn/public_html/wp-content
WP_PLUGIN_DIR: /home/rw3lnlsn/public_html/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8mb4
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

`
```
</details>

**Comment by lauragonzalezz** [comment]
I have the same issue. Plus, I can't copy and paste styles to columns inside inner sections: https://github.com/elementor/elementor/issues/9811#issuecomment-564059438

**Comment by levi-beers** [comment]
Also experiencing both of these issues (duplicate inner column and copy/paste inner styles)

**Comment by KiQdev** [comment]
Also having the same problems as above

**Comment by Bovelett** [comment]
Same here. Same goes for dragging column content into another column in an inner section. The inner columns are immediate invisible upon dragging.

**Comment by robtucker168** [comment]
Same issues here.

**Comment by Worldwebscapes** [comment]
Same as above. 
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.20
	PHP Version: 7.3.12
	PHP Max Input Vars: 1000
	PHP Max Post Size: 2000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.3
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.2.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 5.8.7
		Author: Elliot Condon

	Dynamic Content for Elementor
		Version: 1.8.1.1
		Author: Dynamic.ooo

	Ele Custom Skin
		Version: 1.3.9
		Author: Dudaster.com

	Elementor
		Version: 2.8.0
		Author: Elementor.com

	Elementor Blocks for Gutenberg
		Version: 1.0.1
		Author: Elementor.com

	Elementor Pro
		Version: 2.7.3
		Author: Elementor.com

	Popup Trigger URL for Elementor Pro
		Version: 1.0.1
		Author: Suki WordPress Theme

	Simple Custom CSS and JS
		Version: 3.28
		Author: SilkyPress.com

	WooCommerce
		Version: 3.8.1
		Author: Automattic

	WooCommerce Admin
		Version: 0.23.1
		Author: WooCommerce

	WP User Avatar
		Version: 2.2.4
		Author: flippercode

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.8.1

Feel free to update

Thanks!

**Comment by manoj-identixweb** [comment]
Hi Shilo,

Hope you are well.

Thanks for the new update. I have checked and it's working fine.

Thanks again.
